src/constants/priceConst.js

```diff
@@ -1,5 +1,5 @@
 export const PriceData = {
-  title: `Get in touch with us to calculate prices\nfor your business`,
+  title: `Get in touch with us to calculate prices for your business`,
   text: 'Leave your contacts and request, and our manager will reach out to you to establish our futher partnership!',
```

Comment:
'establish our future partnership'

we should change this to be like the other one, (and it's misspelled anyway)